setup.cfg

```
@@ -1,14 +1,28 @@
 [metadata]
-name = cdk_utils
-version = file: version.txt
+name = cdkutils
+version = attr: cdkutils.__version__
 author = Met Office (Transport Tribe)
 author_email = techsolutionsdeliverybgteam@metoffice.onmicrosoft.com
-description =expected_parameter_keys  aws-utils library used to test aws infrastructure across all BG products
+description =cdk-utils library used with CDK to support the deployment of Transport Tribe applications.
 url = https://github.com/MetOffice/bg-aws-utils
 [options]
 packages = find:
 package_dir= =src
 install_requires =
-    boto3
+    pygit2==1.5.0
+    aws-cdk.core==1.107.0
+    aws-cdk.pipelines==1.107.0
+    boto3==1.17.27
```

Pinning dependencies in a library can cause dependency conflicts in the code that uses it (imagine you're working on code that uses 2 libraries, and each library has pinned its dependencies to different versions). For reusable libraries, it's best to declare a range of supported versions and leave the final decision on pinning to the end user (i.e. whoever is developing the code that makes use of this library)

```suggestion
    pygit2>=1.5.0
    aws-cdk.core>=1.107.0
    aws-cdk.pipelines>=1.107.0
    boto3>=1.17.27
```